### PR TITLE
[ETFE-3636] - Added NRS payload for explain shortage/excess

### PR DESCRIPTION
**Pull request by LewisMitchell**

### CODE REVIEW

**Comment by LewisMitchell** [comment]
Waiting on #142

File: app/uk/gov/hmrc/emcstfe/controllers/SubmitExplainShortageExcessController.scala [hunk 1]
```diff
@@ -41,14 +47,23 @@ class SubmitExplainShortageExcessController @Inject()(cc: ControllerComponents,
     implicit request =>
       withJsonBody[SubmitExplainShortageExcessModel] {
         submission =>
-          if (isEnabled(SendToEIS)) {
-            service.submitViaEIS(submission).map(handleResponse(_))
+          val nrsSubmissionModel = ExplainShortageExcessNRSSubmission.apply(submission, ern)
```
Comment (MJCallahanPage) [inline app/uk/gov/hmrc/emcstfe/controllers/SubmitExplainShortageExcessController.scala:50]: This only needs to be created inside the `IF` statement

**Review by MJCallahanPage — CHANGES_REQUESTED** [review]
One small change.

**Review by MJCallahanPage — APPROVED** [review]
LGTM 🟢

**Pull request merged**